.github/workflows/build_source_release.yml

```diff
@@ -0,0 +1,39 @@
+name: Build Source Release
+
+# Trigger whenever a release is created
+on:
+  release:
+    types:
+      - created
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+
+    - uses: actions/checkout@v1
+      with:
+        submodules: true
+
+    - name: archive
+      id: archive
+      run: |
+        VERSION=$(printf "%s\n" ${{ github.event.release.tag_name }} | sed 's/^v//')
+        PKGNAME="ppsspp-$VERSION"
+        mkdir -p /tmp/$PKGNAME
+        mv * /tmp/$PKGNAME
+        mv /tmp/$PKGNAME .
+        TARBALL=$PKGNAME.tar.xz
+        tar cJf $TARBALL $PKGNAME
+        echo "::set-output name=tarball::$TARBALL"
+
+    - name: upload tarball
+      uses: actions/upload-release-asset@v1
```

Comment:
nit: It might be better to use `svenstaro/upload-release-action@v2` since `actions/upload-release-asset@v1` is unmaintained... though I haven't used the svenstaro one and I'm sure it takes different `with` values.

-[Unknown]

Reply:
that's perfect. no longer maintained means it just keeps on working the way it always did.

Reply:
In my world, "no longer maintained" means that as soon as updates to GitHub Actions come out or the runners move to a new Ubuntu release, it may be broken and no one's gonna fix it.  Likely we would just not even notice, and we don't make many releases per year.

-[Unknown]

Reply:
they wouldn't just break this action, as everybody is using it. i've seen dozens of actions using this one, and nobody svenstaro's (which i happen to know from dwarf fortress times). anyway should it ever break, i volunteer for fixing the upload.